lighthouse-core/report/html/renderer/report-renderer.js

```diff
@@ -247,10 +247,25 @@ class ReportRenderer {
     reportSection.appendChild(this._renderReportFooter(report));
 
     const reportFragment = this._dom.createFragment();
+
     if (!this._dom.isDevTools()) {
       const topbarDocumentFragment = this._renderReportTopbar(report);
       reportFragment.appendChild(topbarDocumentFragment);
     }
+
+    if (scoreHeader && !this._dom.isDevTools()) {
+      const stickyHeader = this._dom.createElement('div', 'lh-sticky-header');
+      this._dom.createChildOf(stickyHeader, 'div', 'lh-highlighter');
+
+      // The sticky header is just the score gauges, but styled to be smaller. Just
+      // clone the gauges from the score header.
+      for (const gaugeWrapperEl of this._dom.findAll('.lh-gauge__wrapper', scoreHeader)) {
```

Comment:
this feels pretty squicky to me :)

Can we make a `renderScoresHeader()` method and use it in both places?

Reply:
done